src/base/bittorrent/sessionimpl.cpp

```diff
@@ -1752,8 +1733,8 @@ lt::settings_pack SessionImpl::loadLTSettings() const
     if (isQueueingSystemEnabled())
     {
         // Internally increase the queue limits to ensure that the magnet is started
```

Comment:
No longer applies and the empty space 3 lines below is pointless now.

Reply:
Thank you 👍 
Fixed.